src/chrome/content/rules/DecorAdventures.com.xml

```diff
@@ -0,0 +1,11 @@
+<!--
+	Active mixed content:
+		- www.decoradventures.com
```

Comment:
Do you have an example?

Reply:
Can't find any MCB, I guess I confused this ruleset with another one.
Thanks for the heads up, removing the `mixedcontent` attribute.